lms/djangoapps/discussion_api/api.py

```diff
@@ -110,6 +110,27 @@ def _is_user_author_or_privileged(cc_content, context):
     )
 
 
+def _update_vote_count(voted, serializer):
+    """
+    Updates the vote_count in the given serializer
+
+    If the user has already voted and `voted == True` do nothing,
```

Comment:
We check the serializer for the `voted` field. If the user has already voted, and a `voted == True` is sent, nothing happens.